### DISCUSSION TITLE
Using meta bind input value inside code block

### DISCUSSION THREAD

**Discussion by kashish52** [mProjectsCode/obsidian-meta-bind-plugin#398] [Q&A]
I have a lot of scripts and code snippets that change based on the user input. I have created an input example below:

To deploy branch `VIEW[{Branch}]`:

```bash
cd app
git pull
make deploy VIEW[{Branch}]
```

Now i want to use this value inside a code block how can i achieve this

**Comment by mProjectsCode** [comment]
There are a few options, the easiest is probably using a js view field and constructing the entire code block via JS.
